Cargo.toml

```diff
@@ -43,14 +43,25 @@ url = { version = "2.2.2", features = ["serde"] }
 impl-serde = "0.3.2"
 regex = "1.5.4"
 
-# dependencies for optional extrinsics feature
-async-std = { version = "1.10.0", optional = true }
-sp-core = { version = "2.0.1", optional = true }
-subxt = { version = "0.14.0", package = "substrate-subxt", optional = true }
-futures = { version = "0.3.19", optional = true }
-hex = { version = "0.4.3", optional = true }
-
-funty = "2.0.0"
+# dependencies for extrinsics (deploying and calling a contract)
+async-std = { version = "1.10.0", features = ["attributes", "tokio1"] }
+ink_metadata = { version = "3.0.0-rc7", features = ["derive"] }
+ink_env = "3.0.0-rc7"
+scale-info = { version = "1.0.0", default-features = false, features = ["derive"] }
+sp-arithmetic = "4.0.0"
+sp-core = "4.0.0"
+sp-keyring = "4.0.0"
+sp-runtime = "4.0.0"
+pallet-contracts-primitives = { package = "pallet-contracts-primitives", git = "https://github.com/paritytech/substrate/", branch = "master" }
```

Comment:
Now other `sp-*` crates are released, we need to release `pallet-contracts-primitives`, and also `sp-rpc` on which it depends. /cc @athei

Reply:
We don't really do releases. Can't you stick with a git ref for everything?

Reply:
Not if we want to release `cargo-contract` to `crates.io`. But that is not an absolute requirement for now.

The alternative would be to copy those types we need over here, or to implement https://github.com/paritytech/subxt/issues/416

Reply:
Released the crate. You can switch to the released version now. Also make sure to bump the rest of the substrate versions.

Reply:
Thank you very much :pray: 

We also need a `5.0.0` of `sp-keyring`: https://github.com/paritytech/substrate/pull/10790